OasysUnits.NanoFramework/GeneratedCode/AreaDensity/UnitsNet.NanoFramework.AreaDensity.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
   <metadata>
     <id>OasysUnits.nanoFramework.AreaDensity</id>
-    <version>5.12.0</version>
+    <version>1.1.1</version>
```

Comment:
why from 5.12 -> 1.1? 
the same for other places

Reply:
@DominikaLos 5.12.0 is the UnitsNet version, it should have been 1.1.0 instead of 5.12.0 before..